gen/KubernetesWatchGenerator/Program.cs

```diff
@@ -150,7 +101,7 @@ static void ToXmlDoc(RenderContext context, IList<object> arguments, IDictionary
                     {
                         if (!first)
                         {
-                            context.Write(Environment.NewLine);
+                            context.Write("\n");
```

Comment:
Shouldn't we keep the `Environment.NewLine` ?

Reply:
it should be `eol` of the generated files, `IKubernetes.Watch.cs`, `Kubernetes.Watch.cs` and `ModelExtensions.cs`.

those files are generated using `lf` already.
and should be generated using `lf` only as `.editorconfig` defined the rule `eol = lf`

Reply:
ok, thanks.